module2.js

```diff
@@ -52,7 +61,10 @@ function question2(carObj) {
 
 function question3(dogs) {
   // TODO
-  return /* ??? */
+  var dog3WeeksOld = dogs.filter(dog => dog.weeksOld > 3);
+  return dogs3WeeksOld.map(dog => dog.name);
```

Comment:
Francisca you were so close! Watch out for typos - also when you run the test it gives you a clue about what's wrong, the error is ` ReferenceError: dogs3WeeksOld is not defined`

Reply:
🤦‍♀️